libs/libc/stdio/lib_getdelim.c

```diff
@@ -53,7 +53,7 @@
  */
 
 #undef HAVE_GETLINE
-#if defined(CONFIG_EOL_IS_CR)
+#if defined(CONFIG_EOL_IS_CR) || defined(CONFIG_EOL_IS_EITHER_CRLF)
```

Comment:
Does this work?  Can getline() handle two character EOL sequences?

Reply:
Yes, the processing logic just follows other similar code in apps cle, see:
apps/system/cle/cle.c:
`1108         /* Newline terminates editing.  But what is a newline? */
1109 
1110 #if defined(CONFIG_EOL_IS_CR) || defined(CONFIG_EOL_IS_EITHER_CRLF)
1111         case '\r': /* CR terminates line */
1112 `

i think in either case we can handler by CR in geline by default

Reply:
I don't think so.  The /n will not be consumed but will come through as data, won't it?

Reply:
Ok, I will consider a better solution,  new change will upload later, please close this PR, thanks for review!